dll/win32/advapi32/advapi32.spec

```
@@ -1,7 +1,10 @@
 1 stdcall I_ScGetCurrentGroupStateW(ptr wstr ptr)
-@ stdcall A_SHAFinal(ptr ptr)
-@ stdcall A_SHAInit(ptr)
-@ stdcall A_SHAUpdate(ptr ptr long)
+@ stdcall -version=0-0x502 A_SHAFinal(ptr ptr)
```

```suggestion
@ stdcall -version=0x502 A_SHAFinal(ptr ptr)
```
as ReactOS is not built for XP and older.

---

Committed.